LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018-2021, the PyBaMM team.
+Copyright (c) 2022, the PyBaMM team.
```

Comment:
@tinosulzer I tried running the `FantasticFiasco/action-update-license-year@v2` action on my fork and it is creating a range of years instead of a single year (See https://github.com/priyanshuone6/PyBaMM/pull/22/files). Should we include a single year or a range of years?

Reply:
Yeah let's do the range. I should have kept it as a range